app/docs/0.12.x/healthchecks.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Health Checks Reference
+---
+
+# Health Checks Reference
+
+You can make an API proxied by Kong use a [ring-balancer][1], configuring
+by adding an [upstream][2] entity that contains one or more [target][3]
+entities, each target pointing to a different IP address (or hostname) and
+port. The ring-balancer will balance load among the various targets, and based
+on the [upstream][2] configuration, will perform health checks on the targets,
+making them as healthy or unhealthy whether they are responsive or not. The
+ring-balancer will then only route traffic to healthy targets.
+
+Kong supports two kinds of health checks, which can be used separately or in
+conjunction:
+
+* **active checks**, where a specific endpoint in the target is periodically
+requested and the health of the target is determined based on its response;
+
+* **passive checks** (also known as **circuit breakers**), where Kong analyzes
+the ongoing traffic being proxied and determines the health of targets based
+on their behavior responding requests.
+
+[1]: /docs/{{page.kong_version}}/loadbalancing#ring-balancer
+[2]: /docs/{{page.kong_version}}/loadbalancing#upstream
+[3]: /docs/{{page.kong_version}}/loadbalancing#ring-balancer
+
+### Table of Contents
+
+- [Healthy and unhealthy targets](#healthy-and-unhealthy-targets)
+- [Active health checks](#active-health-checks)
+- [Passive health checks (circuit breakers)](#passive-health-checks-circuit-breakers)
+- [Summary of pros and cons](#summary-of-pros-and-cons)
+
+### Healthy and unhealthy targets
+
+The objective of the health checks functionality is to dynamically mark
+targets as **healthy** or **unhealthy**, **for a given Kong node**. There is
```

Comment:
visual style, would avoid bolding the phrase 'for a given Kong node'. i understand the value of emphasis here, but it loses its meaning placed next to other bold values.

Reply:
Good point. I'll remove the bolding of the other ones. :)